### ISSUE TITLE
GBC Rendering Issue : Doc Cosmos

### ISSUE THREAD

**Issue by pmprog** [mgba-emu/mgba#2926]
Tried running Doc Cosmos on mGBA, and the scrolltext doesn't scroll cleanly

Video of what it should look like: https://www.youtube.com/watch?v=ILjB59myE_M

Recording from mGBA (v0.10.2)
![doc](https://github.com/mgba-emu/mgba/assets/1705629/369ad28a-4702-4854-b260-30084aec96b1)

Does the same on my Android RetroArch running v0.9.2

https://www.bitmapsoft.co.uk/product/doc-cosmos-gbc/

**Comment by TiKevin83** [comment]
Can you replicate this behavior in Gambatte Speedrun or SameBoy or the Gambatte core within BizHawk, and does this happen on an actual GBA?

**Comment by pmprog** [comment]
No, the scrolling is smooth and correct on a physical GBA, and Sameboy running in the same Android RetroArch

**Comment by pmprog** [comment]
Works fine in 0.15.8 in flatpak
